playbooks/robusta_playbooks/common_actions.py

```diff
@@ -0,0 +1,28 @@
+from robusta.api import *
+
+
+class FindingOverrides(ActionParams):
+    title: str = ""
+    description: str = ""
+    severity: str = ""
```

Comment:
Is it possible to define this as a FindingSeverity? E.g. `severity: Optional[FindingSeverity] = None` or something like that?

Reply:
no easy way to initialize it by enum key